packages/cli/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @redocly/cli
 
+## 1.5.0
+
+### Minor Changes
+
+- Added new rule `array-parameter-serialization` to require that serialization parameters `style` and `explode` are present on array parameters.
+
+### Patch Changes
+
+- Support standalone docs development to facilitate more contributors, add docs quality checks to CI.
```

Comment:
I included it because it affects any external contributors on this public project. I agree that it doesn't affect users though, so I don't mind if you want to drop it.

Reply:
I think I will drop it before the release.